package.json

```
@@ -42,5 +42,8 @@
     "posttest": "npm run postpublish",
     "svgo": "svgo --config=./.svgo.yml",
     "get-filename": "node scripts/get-filename.js"
+  },
+  "engine": {
+    "node": ">=>=0.12.18"
```

I think this should be:

```suggestion
    "node": ">=0.12.18"
```

---

What a mistake :sweat_smile: Sorry, it's fixed now. I will investigate these features before merge.

---

- Getters are supported in 0.12.18 (tested manually and defined in [node.green](https://node.green/#ES2017-annex-b-Object-prototype-getter-setter-methods-__lookupGetter__)).
- `enumerable` property was been added at the beggining of the [`Object.defineProperty` support](https://node.green/#ES2017-annex-b-Object-prototype-getter-setter-methods-__lookupGetter__). More info [here](https://www.ecma-international.org/ecma-262/6.0/#sec-property-attributes).